### PR TITLE
Pass init options from MultiProcessConsumer down to SimpleConsumer

### PR DESCRIPTION
**Pull request by mahall**
Fixes mumrah/kafka-python#248.

### CODE REVIEW

File: kafka/consumer/multiprocess.py [hunk 1]
```diff
@@ -103,7 +107,8 @@ class MultiProcessConsumer(Consumer):
     def __init__(self, client, group, topic, auto_commit=True,
                  auto_commit_every_n=AUTO_COMMIT_MSG_COUNT,
                  auto_commit_every_t=AUTO_COMMIT_INTERVAL,
-                 num_procs=1, partitions_per_proc=0):
+                 num_procs=1, partitions_per_proc=0,
+                 simple_consumer_options={}):
```
Comment (sontek) [inline kafka/consumer/multiprocess.py]: A mutable dict as a kwarg has import level sideeffects, it'll be one dict shared across all instances.  Please assign this as `None` and then check inside the function.

**Comment by sontek** [comment]
@mahall This doesn't merge cleanly.  Can you get it working again?

**Comment by dpkp** [comment]
#336 improves on this a bit -- plan to merge that and close this unless there are strong objections.

**Comment by dpkp** [comment]
#336 merged -- thanks for your work on this issue!

**Pull request closed without merging**